themes/default-bootstrap/modules/blocknewsletter/blocknewsletter.tpl

```diff
@@ -32,6 +32,7 @@
                 <button type="submit" name="submitNewsletter" class="btn btn-default button button-small">
                     <span>{l s='Ok' mod='blocknewsletter'}</span>
                 </button>
+				{hook h='displayGDPRConsent' mod='psgdpr' id_module=$id_module}
```

Comment:
@clementdaubeuf ekhm, is this variable available?

Reply:
the variable is added in this pull request : https://github.com/PrestaShop/blocknewsletter/pull/29

Reply:
You follow the way for The Official GDPR Compliance Module by PrestaShop but the other modules ?

Reply:
Can you use spaces  instead of tabs please :)